### PR TITLE
Add security section to issue template

### PR DESCRIPTION
**Pull request by mikaelm12**
We had this in the SignalR repo to prevent folks from making public bugs with security issues.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ name: Bug report
 about: Create a report about something that is not working
 ---
 
+### If you believe you have an issue that affects the security of the platform please do NOT create an issue and instead email your issue details to secure@microsoft.com. Your report may be eligible for our [bug bounty](https://technet.microsoft.com/en-us/mt764065.aspx).
```
Comment (anurse) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
### If you believe you have an issue that affects the security of the platform please do NOT create an issue and instead email your issue details to secure@microsoft.com. Your report may be eligible for our [bug bounty](https://technet.microsoft.com/en-us/mt764065.aspx) but ONLY if it is reported through email.
```

**Review by blowdart — APPROVED** [review]
Finally :)

**Comment by mikaelm12** [comment]
@aspnet-hello triage https://dev.azure.com/dnceng/public/_build/results?buildId=159217&view=logs

**Comment by aspnet-hello** [comment]
This comment was made automatically. If there is a problem contact ryanbrandenburg.

I've triaged the above build.

**Comment by mikaelm12** [comment]
Should I go ahead and add this to EF as well?

**Review by Eilon — APPROVED** [review]

**Comment by Eilon** [comment]
> Should I go ahead and add this to EF as well?

Yes, and also Extensions repo.

**Pull request merged**